tests/usm/usm_atomic_access.h

```diff
@@ -56,15 +56,15 @@ struct run_all_tests {
   void operator()(sycl::queue &queue, sycl_cts::util::logger &log,
                   const std::string &typeName) {
     if constexpr ((sizeof(CounterT) * CHAR_BIT) == 64) {
-      if (!UseAtomic64Flag::value) {
+      if constexpr (!UseAtomic64Flag::value) {
         return;
       }
       if (!queue.get_device().has(sycl::aspect::atomic64)) {
         log.note("Device does not perform 64-bit atomic operations");
         return;
       }
     } else {
-      if (UseAtomic64Flag::value) {
+      if constexpr (UseAtomic64Flag::value) {
```

Comment:
What is the goal of this test?
OK, this is for the case we have a counter which has not 64 bits but we still have `UseAtomic64Flag` we cannot use here...
Should we have a logging message here too or just ignoring is good?

Reply:
I think that need logging in case, when we skip test due to some reasons, if all happened as expected log messages is useless